docs/api_reference/src/quantities.tex

```diff
@@ -913,11 +913,22 @@
 a specialization of \tcode{power<$x$, $n$, $d$>}, then its input value is $x^{n/d}$, and
 the following also apply.
 \item
-Otherwise, a type $x$ is an input constant.
+Otherwise, a type $x$ is an input symbolic constant.
 \end{itemize}
 \begin{example}
-The input of \tcode{derived_unit<si::kilo_<si::metre>, per<power<non_si::hour, 2>>>}
-is $\tcode{km}/\tcode{h}^2$.
+Item by item, this algorithm step goes from the \Cpp{} parameter type
+\tcode{derived_unit<si::kilo_<si::metre>, per<power<non_si::hour, 2>>}
+\begin{itemize}
+\item
+to $\tcode{si::kilo_<si::metre>} \times \tcode{per<power<non_si::hour, 2>}$ (product of \tcode{To}'s arguments),
```

Comment:
Maybe a kilogram could be also added in front to present how more than one argument is spelled in a nominator or denominator?

Reply:
I took this from the example in [qty.expr.temp.general].
We could just add it there.

Reply:
Does the updated example in the simplification step suffice to show a side of $/$ with multiple terms?

Updated the example in [qty.expr.temp.general] to
```C++
using namespace si::unit_symbols;
static_assert(
  std::is_same_v<decltype(kg * km / square(h)),
                 derived_unit<si::kilo_<struct si::gram>, si::kilo_<struct si::metre>,
                              per<power<struct non_si::hour, 2>>>>);
```

Reply:
I do not know. It is a tradeoff between being terse in examples and showing a few features at once. We can use an even more complex unit here, but the example will be longer.

Reply:
These *elaborated-type-specifier*s are really offputting.
They don't appear in error messages and [printed output](https://godbolt.org/z/veEh5Gn8G).
Users don't write these.
The implementation also doesn't write these.
But we are presenting them in these examples.
How can we do better?

Reply:
I do not know what you mean 😢 Could you please provide a specific example of what is wrong?

Reply:
> We do not write the latter, so it feels wrong to have examples using it.

Totally agree. We always claim that the user always uses values and never deals with types. Providing examples that may suggest otherwise is not useful and may make our naming conventions even more controversial.